addons/overheating/functions/fnc_jamWeapon.sqf

```diff
@@ -35,25 +36,32 @@ private _temp = 1 max (_unit getVariable [format [QGVAR(%1_temp), _weapon], 0]);
 private _jamTypesAllowed = getArray (configFile >> 'CfgWeapons' >> currentWeapon _unit >> QGVAR(jamTypesAllowed));
```

Comment:
I understand this is already existing, but where does `currentWeapon _unit` come from - shouldn't this file be exclusively be using `_weapon`?

Reply:
Yeah probably should be!